third_party/packages/path_parsing/README.md

```diff
@@ -3,3 +3,12 @@
 Split from the Flutter path drawing library to create a pure Dart parsing
 library for SVG paths and code generation (without dependencies on Flutter
 runtime).
+
+## Commemoration
+
+This package was originally authored by
+[Dan Field](https://github.com/dnfield) and has been forked here
+from https://github.com/dnfield/dart_path_parsing. Dan was a member of the
```

Comment:
Just noticed this while duplicating: how about `[dnfield/dart_path_parsing](http://github.com/dnfield/dart_path_parsing)` so the link isn't as clunky in the rendered text?

Reply:
Good idea. Done.